packages/shared/src/styles/constants.scss

```diff
@@ -24,7 +24,6 @@ $color-red-3: #a32430;
 $color-red-4: #d33636;
 $color-red-5: #eb3e48;
 $color-red-6: #ec3f3f29;
-$color-red-6: #ffe1e3;
```

Comment:
rename it to` $color-red-7` and use it in `packages/appstore/src/components/add-options/add-options.scss` instead of `color-red-6`